indexer/feature.cpp

```diff
@@ -806,6 +806,12 @@ string const & FeatureType::GetHouseNumber()
   return m_params.house.Get();
 }
 
+bool const FeatureType::HasStreetName()
+{
+  ParseCommon();
+  return !m_params.name.IsEmpty();
```

Comment:
Its a name of a feature, its not street name...

AFAIK each feature does not include full address info, only a house number.

IDK how they're associated, @vng knows.

Reply:
Yes, it is not about the street name and usage is not correct respectively.

Reply:
Oh, I see. Then I don't know how to implement this.